src/webview-js/toc-editor/toc-editor.jsx

```diff
@@ -0,0 +1,155 @@
+import { h, Fragment, render } from 'preact';
+import { useEffect, useState } from 'preact/hooks';
+import 'react-sortable-tree/style.css';
+import SortableTree from 'react-sortable-tree';
+
+const vscode = acquireVsCodeApi();
+const nodeType = 'toc-element'
+
+const removeExpanded = (obj) => {
```

Comment:
Any chance this could be a `.tsx` file instead? It might be overkill but it would be easier to copy/paste later if we make more complicated views.

Reply:
I just tried it to see if I could do it in a reasonable period of time. I'm not sure I can. I can make a card

Reply:
card created